packages/KModal/KModal.vue

```diff
@@ -16,7 +16,8 @@
           <div
             v-if="$scopedSlots.title || !hideTitle"
             class="k-modal-header modal-header mb-5"
-            role="heading">
+            role="heading"
+            aria-level="1">
```

Comment:
I'm not sure this is necessary or that it will even be picked up since there is an `aria-label` on the parent `.k-modal` element. (See the warning [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role#accessibility_concerns)).

Either way, the value should be lower than 1 since it isn't the page title.
> A level 1 heading, indicated with `aria-level="1"`, usually indicates the main heading of a page

Reply:
@adamdehaven that's a good point, I didn't notice the `aria-label` with the same value. I think it should be fine if we just remove the `role` and `aria-level` attribute?

Reply:
You can leave the `role` as that will still call out the text is a heading to screen readers; however, you can remove the `aria-level` attribute.

> If there is more than one heading on the page and the heading hierarchy is defined through the visual presentation, the aria-level attribute should be used to indicate the hierarchical level of the heading.

Since this is the case, the heading is the only one inside the modal (and it matches the label) meaning it wouldn't need a level. 

Really, we should be using `aria-labeledby` as [shown here](https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/dialog.html), but we have an open ticket to improve accessibility that will cover a lot of this.

Reply:
There are some asks for a clean aXe report in the portal, which would likely require having the `aria-level` attribute present. I think for the time being before the change to use `aria-labeledby`, we can set this `aria-level` to 2?

Reply:
`aria-level` defaults to `2` if not provided

Reply:
The aXe tool will throw the warning if it's not explicitly provided - would it be okay to include it?

Reply:
Sure -- I'm thinking maybe with a value of `3` since it's a modal, but if you want to stick with `2` that's fine as well. Don't forget the test snapshots will need to be updated as well.